specifications/active/thincit.yml

```diff
@@ -1,4 +1,7 @@
 name: THINCIT
+vendor: LUNDBECK
+model: THINCIT
+version: v1
```

Comment:
semantic versioning would be preferable. This could be `1.0.0`

Reply:
Good idea. For all source-types i assume. Will change them